### PR TITLE
Aqcu 1639: add publish flag to field visit measurements

### PR DESCRIPTION
**Pull request by mhines-usgs**

### CODE REVIEW

File: src/test/java/gov/usgs/aqcu/builder/ReportBuilderServiceTest.java [hunk 1]
```diff
@@ -638,20 +638,20 @@ public void getFieldVisitMeasurementTest() {
 						.setDisplay("20.0090").setNumeric(Double.valueOf("20.0090")))
 				.setMeanGageHeight((QuantityWithDisplay) new QuantityWithDisplay().setUnit("meanGageHeightUnits")
 						.setDisplay("2.0090").setNumeric(Double.valueOf("2.0090")))
-				.setMeasurementStartTime(nowInstant));
+				.setMeasurementStartTime(nowInstant)
+				.setPublish(false));
 		FieldVisitMeasurement expected = new FieldVisitMeasurement("20.0090", new BigDecimal("20.0090"),
-				new BigDecimal("21.00945000"), new BigDecimal("19.00855000"), nowInstant);
-		assertThat(actual, samePropertyValuesAs(expected));
```
Comment (mhines-usgs) [inline src/test/java/gov/usgs/aqcu/builder/ReportBuilderServiceTest.java]: this is a weird one, not sure why it fails using one kind of compare versus another

**Comment by mhines-usgs** [comment]
There are example data from these changes on the JIRA task

**Comment by zmoore-usgs** [comment]
@mhines-usgs Is this ready to be merged?

**Comment by mhines-usgs** [comment]
@zmoore-usgs thanks for the heads up. i bet chuck was confused ;)

**Pull request merged**